express/routes/index.js

```diff
@@ -0,0 +1,35 @@
+const bodyParser = require('body-parser');
+const cors = require('cors');
+const express = require('express');
+const fs = require('fs');
+const path = require('path');
+
+const errorHandler = require('../middleware/error-handler');
+
+const basename = path.basename(__filename);
+
+const app = express();
+
+app.use(cors());
+
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({
+    extended: true
+}));
+
+app.get('/', (req, res) => {
+    res.sendStatus(200);
+});
+
+fs
+    .readdirSync(__dirname)
+    .filter(file => {
+        return (file.indexOf('.') !== 0) && (file !== basename) && (file.slice(-3) === '.js');
+    })
+    .forEach(file => {
+        app.use(`/${file.slice(0, -3)}`, require(path.join(__dirname, file)));
+    });
```

Comment:
Seems that a lot of things are mixed up here, can we separate them to different files?